docs/tutorials/multimodal/object_detection/inference/detection_inference_coco.md

```diff
@@ -0,0 +1,166 @@
+# AutoMM Detection - Inference with Pretrained Faster R-CNN on VOC Format Dataset
+:label:`sec_automm_detection_infer_coco`
+
+In this section, we show an example to run inference COCO dataset in COCO Format. 
+Different from running evaluation, the purpose is to get detection results for potential down-stream tasks.
+
+[//]: # (In this section, our goal is to evaluate Faster-RCNN model on VOC2007 dataset in VOC format.)
+
+[//]: # (See \[Convert VOC to COCO] for how to quickly convert a VOC format dataset.)
+
+[//]: # (In previous section :ref:`sec_automm_detection_eval_fasterrcnn_coco`, we evaluated Faster-RCNN on COCO dataset.)
+
+[//]: # (We strongly recommend using COCO format, but AutoMM still have limited support for VOC format for quick proof testing.)
+
+## Prepare data
+For running this tutorial, you should have COCO dataset prepared.
+If you haven't already, head over to :label:`sec_automm_detection_prepare_coco17` to learn how to get COCO dataset.
+This tutorial assumes that COCO dataset is under `~/data`, i.e. it is located in `~/data/coco17`
+
+## Creating the `MultiModalPredictor`
+To start, import MultiModalPredictor:
+```python
+from autogluon.multimodal import MultiModalPredictor
+```
+### Use a pretrained model
+You can download a pretrained model and construct a predictor with it. 
+In this example, we use the VFNet with ResNext as backbone and Feature Pyramid Network (FPN) as neck.
+
+```python
+checkpoint_name = "vfnet_x101_64x4d_fpn_mdconv_c3-c5_mstrain_2x_coco"
+num_gpus = -1  # using all GPUs if available
+```
+You can also use other model by setting `checkpoint_name` to other names. 
+Please refer to :ref: `selecting_models` for details about model selection.
+
+As before, we create the MultiModalPredictor with selected checkpoint name and number of GPUs.
+We also need to specify the `problem_type` to `"object_detection"`.
+
+```python
+predictor = MultiModalPredictor(
+    hyperparameters={
+        "model.mmdet_image.checkpoint_name": checkpoint_name,
+        "env.num_gpus": num_gpus,
+    },
+    problem_type="object_detection",
+)
+```
+
+### Use a finetuned model
+You can also use a previously trained/finetuned predictor to run inference with.
+First specify the predictor path, for example:
+```python
+load_path = "./AutogluonModels/ag-20221104_185342"
+```
+Then load the predictor:
+```python
+predictor = MultiModalPredictor.load(load_path)
+```
+
+## Setting up data
+
+For COCO format data, we need to provide the path for the data split used for inference.
+
+```python
+test_path = "~/data/coco17/annotations/instances_val2017.json"
+```
+
+## Running inference
+To run inference, perform:
+
+```python
+pred = predictor.predict(test_path)
+```
+
+To run inference and save results, run the following:
+```python
+pred = predictor.predict(test_path, save_results=True)
```

Comment:
Let's mention by default, we don't save results. But if you want results, you can set `save_results=True`, and you will get a .txt file looking like xxx.